int8 IN_PROGRESS=0
int8 COMPLETE=1
int8 FAILED=-1

string action_id
uuid_msgs/UniqueID uuid
int8 action_status